cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -51,9 +58,104 @@ def transform_survey_responses_generic(df: DataFrame) -> DataFrame:
     df = convert_barcode_null_if_zero(df, "blood_sample_barcode")
     df = assign_taken_column(df, "swab_taken", reference_column="swab_sample_barcode")
     df = assign_taken_column(df, "blood_taken", reference_column="blood_sample_barcode")
+    # df = assign_true_if_any(
+    #    df=df,
+    #    column_name_to_assign="sympt_covid_cghfevamn",
+    #    reference_columns=[
+    #        "sympt_covid_cough",
+    #        "sympt_covid_fever",
+    #        "sympt_covid_loss_of_smell",
+    #        "sympt_covid_loss_of_taste",
+    #    ],
+    #    true_false_values=["Yes", "No"],
+    # )
+    # df = assign_true_if_any(
+    #     df=df,
+    #     column_name_to_assign="any_symptoms_last_7_days_or_now",
+    #     reference_columns=["symptoms_last_7_days_any", "think_have_covid_symptoms_now"],
+    #     true_false_values=["Yes", "No"],
+    # )
+    # df = assign_true_if_any(
+    #     df=df,
+    #     column_name_to_assign="symptoms_last_7_days_cghfevamn_symptom_group",
+    #     reference_columns=[
+    #         "symptoms_last_7_days_cough",
+    #         "symptoms_last_7_days_fever",
+    #         "symptoms_last_7_days_loss_of_smell",
+    #         "symptoms_last_7_days_loss_of_taste",
+    #     ],
+    #     true_false_values=["Yes", "No"],
+    # )
+    df = assign_true_if_any(
+        df=df,
+        column_name_to_assign="think_have_covid_cghfevamn_symptom_group",
+        reference_columns=[
+            "symptoms_since_last_visit_cough",
+            "symptoms_since_last_visit_fever",
+            "symptoms_since_last_visit_loss_of_smell",
+            "symptoms_since_last_visit_loss_of_taste",
+        ],
+        true_false_values=["Yes", "No"],
+    )
+    # df = assign_any_symptoms_around_visit(
+    #     df=df,
+    #     column_name_to_assign="any_symptoms_around_visit",
+    #     symptoms_bool_column="any_symptoms_last_7_days_or_now",
+    #     id_column="participant_id",
+    #     visit_date_column="visit_date",
+    #     visit_id_column="visit_id",
+    # )
+    # df = assign_any_symptoms_around_visit(
+    #     df=df,
+    #     column_name_to_assign="symptoms_around_cghfevamn_symptom_group",
+    #     id_column="participant_id",
+    #     symptoms_bool_column="sympt_now_cghfevamn",
```

Comment:
This is a stata name, needs updating to pipeline name:

```suggestion
    #     symptoms_bool_column="symptoms_last_7_days_cghfevamn_symptom_group",
```